### DISCUSSION TITLE
Hover on Card

### DISCUSSION THREAD

**Discussion by giladvidal** [UI5/webcomponents-react#2930] [Q&A] [answered]
Hello,
I have a card in the page. When moving the cursor from the page area to the card area, the card frame is being highlighted. How is it possible to have the card as part of the page? 

<Card style={{ 
        marginTop: '20px',
    }}>
    ...
    <AnalyticalTable
        className="cursor-text"
        ...
    </AnalyticalTable>
</Card>
 
![image](https://user-images.githubusercontent.com/56625024/167581923-3161a2ea-e7c2-4bd2-836e-f2e4884dad36.png)

Thanks,
Gilad

**Comment by Lukas742** [comment]
Hi @giladvidal 

I'm not quite sure what you want to achieve. If you want to add a `border-color` or `box-shadow` to the Card while the user is hovering over it, you can do that by using `onMouseOver` and `onMouseOut` ([codeSandbox](https://codesandbox.io/s/heuristic-germain-wqbe8s?file=/src/App.js)). If that is not the case then please elaborate a bit more.

Edit:
I just noticed that you don't even need js for this... You can just use the `:hover`selector ([codeSandbox](https://codesandbox.io/s/competent-joana-h386p4?file=/src/App.js))

**Comment by giladvidal** [comment]
Thanks @Lukas742 , you are a real value to the company :)

Screenshot to explain better.
![image](https://user-images.githubusercontent.com/56625024/167654184-7f706e26-b400-44a2-bacb-88441a348e4d.png)

**Comment by Lukas742** [comment, marked as answer]
Thanks @giladvidal :) 

I overlooked that you are using the "horizon" theme. With horizon the `box-shadow` on hover is integrated by design and should preferably not be changed.  If you still want to change it you could overwrite the hover CSS variable to the default one:
```
'--_ui5_card_hover_box_shadow': 'var(--_ui5_card_box_shadow)'
```
Keep in mind though, that these are internal variables and that they can be changed without an entry in the change log of UI5 Web Components.

If you feel like altering the styles of the card root should be possible without overwriting CSS variables then you could also open a [feature request](https://github.com/SAP/ui5-webcomponents/issues/new/choose) in the UI5 Web Components repo and ask our colleagues if they could add the [::part pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/::part) to the element in the shadow root.
